src/components/MessageList.vue

```diff
@@ -234,7 +234,10 @@ export default {
             return highlightFound;
         },
         nickStyle: function nickColour(nick) {
-            return 'color:' + TextFormatting.createNickColour(nick) + ';';
+            if (state.setting('buffers.colour_nicknames_in_messages')) {
```

Comment:
This should use `this.bufferSetting('colour_nicknames_in_messages')` so that per-buffer settings are possible.